assemble_installer.py

```diff
@@ -22,10 +48,50 @@
             subprocess.call(f"nix-store -r {installer_url}", shell=True)
         installers.append((installer_url, system))
     else:
+        print(
+            f"Build {build_id} not finished. Check status at https://hydra.nixos.org/eval/{hydra_eval['id']}#tabs-unfinished"
+        )
         sys.exit(0)
 
-subprocess.run(["git", "fetch", "origin", "prerelease"], check=True)
-subprocess.run(["git", "checkout", "-b", "prerelease", "origin/prerelease"], check=True)
+with open("Cargo.toml", "rb") as f:
+    cargo_toml = tomllib.load(f)
+version = cargo_toml["package"]["version"]
```

Comment:
I think this is the most-reasonable action atm, but in case someone's reading back over this for context later (I've already gotten a Q from the Nix team about how we're versioning this) I'll caution: 

This tracks the detsys versioning. I think that's very defensible, here. 

It saves us from having to debate a call about how to version this that may or may not mesh with how the Nix team and/or broader org/community want to see this installer versioned relative to Nix and upstream.

This installer may well have a different relationship to Nix than the DetSys upstream does (i.e., we don't know if it'll always iterate with and use the latest Nix, or if it'll use some notion of "stable" nix like detsys and nixpkgs do, etc.)

If you're here from the future: we need something for now, but don't treat this as a well-considered decision if you have good reasons to do something different.